service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandler.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.enums.UserRole;
+import uk.gov.hmcts.reform.fpl.events.AdditionalApplicationsUploadedEvent;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
+import uk.gov.hmcts.reform.fpl.request.RequestData;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.AdditionalApplicationsUploadedEmailContentProvider;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+
+import java.util.List;
+
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class AdditionalApplicationsUploadedEventHandler {
+    private final IdamClient idamClient;
+    private final RequestData requestData;
+    private final NotificationService notificationService;
+    private final HmctsAdminNotificationHandler adminNotificationHandler;
+    private final AdditionalApplicationsUploadedEmailContentProvider additionalApplicationsUploadedEmailContentProvider;
+
+    @EventListener
+    public void notifyAdmin(final AdditionalApplicationsUploadedEvent event) {
+        List<String> roles = idamClient.getUserInfo(requestData.authorisation()).getRoles();
```

Comment:
can be requestData.userRoles() (set instead of list)
see: https://tools.hmcts.net/jira/browse/FPLA-2721

Reply:
then can remove idam client call completely from here + tests

Reply:
Made the change suggested but unable to completely remove from tests as it's still needed for getting the email